components/shared/title-bar.js

```diff
@@ -13,15 +13,16 @@ const styles = {
         width: '100%',
         backgroundColor: '#EAEAEA',
         margin: '0 auto',
-
+        paddingTop: '40px',
     },
 
     body: {
         flexGrow: 1,
-        maxWidth: '1200',
+        maxWidth: '1200px',
         textAlign: 'left',
         padding: '40px 20px',
         margin: '0 auto',
+        position: 'sticky',
     },
```

Comment:
after "body" styles:
line 26: 
~~~
    gridItem: {
        width: '100%'
    },
~~~

for the second occurrence of <Grid>(line 55) change to:
`<Grid item className={classes.gridItem}>`